### ISSUE TITLE
bug: Unable to update stack: 'Conditions' when deploying from Serverless

### ISSUE THREAD

**Issue by smbkr** [localstack/localstack#5447]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I try to deploy my `serverless` stack to localstack 0.14 with the `serverless-localstack` plugin, I get the following error from Serverless:

```
Serverless: Updating Stack...
 
 Serverless Error ----------------------------------------
 
  Unable to update stack "mmh-api-local": 'Conditions'
```

And the following in the logs for the localstack container:

```
localstack_main  | 2022-02-08T16:16:09.649:DEBUG:localstack.services.cloudformation.cloudformation_api: Unable to update stack "mmh-api-local": 'Conditions' Traceback (most recent call last):
localstack_main  |   File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_api.py", line 587, in update_stack
localstack_main  |     deployer.update_stack(new_stack)
localstack_main  |   File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1236, in update_stack
localstack_main  |     self.apply_changes(self.stack, new_stack, stack_name=self.stack.stack_name, action="UPDATE")
localstack_main  |   File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1517, in apply_changes
localstack_main  |     existing_stack.template["Conditions"].update(new_stack.template.get("Conditions", {}))
localstack_main  | KeyError: 'Conditions'
localstack_main  | 
```

### Expected Behavior

It should deploy without errors

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker compose up -d

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    sls deploy --stage local

### Environment

```markdown
- OS: Mac OS 11.6.1
- LocalStack: 0.14
```

### Anything else?

I do not get this error with version 0.13.3. I get the same error using `latest`:

```
❯ docker image inspect localstack/localstack:latest | jq '.[0].RepoDigests'
[
  "localstack/localstack@sha256:dd5c2d8e2158cef0e9be0cce61f9150b74b0c5e658fec4fa056b5535af858817"
]
```

**Comment by smbkr** [comment]
Possibly related to #5063 but there's not much detail there, so unsure.

**Comment by whummer** [comment]
Thanks for reporting @smbkr . This should be fixed in  #5449 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists.. Thanks!
